user/enterprise/ssl-certificate-management.md

```diff
@@ -88,131 +88,167 @@ Upon restart you can then verify whether SSL verification is now working. If you
-cancel): ops@example.com
-```
-
-Then, ***accept the Terms of Services*** and decide if you'd like to share your email address with the EFF:
-
-```
--------------------------------------------------------------------------------
-Please read the Terms of Service at
-https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf. You must agree
-in order to register with the ACME server at
-https://acme-v01.api.letsencrypt.org/directory
--------------------------------------------------------------------------------
-(A)gree/(C)ancel: A
-
--------------------------------------------------------------------------------
-Would you be willing to share your email address with the Electronic Frontier
-Foundation, a founding partner of the Let's Encrypt project and the non-profit
-organization that develops Certbot? We'd like to send you email about EFF and
-our work to encrypt the web, protect its users and defend digital rights.
--------------------------------------------------------------------------------
-(Y)es/(N)o: N
-```
-
-In the last step you're ***providing your domain name:***
-
-```
-Please enter in your domain name(s) (comma and/or space separated)  (Enter 'c'
-to cancel): travis.example.com
-```
-
-After that ***finished successfully,*** you'll see a message similar to the one below:
-
-```
-IMPORTANT NOTES:
- - Congratulations! Your certificate and chain have been saved at:
-   /etc/letsencrypt/live/travis.example.com/fullchain.pem
-   Your key file has been saved at:
-   /etc/letsencrypt/live/travis.example.com/privkey.pem
-   Your cert will expire on 2018-02-07. To obtain a new or tweaked
-   version of this certificate in the future, simply run certbot
-   again. To non-interactively renew *all* of your certificates, run
-   "certbot renew"
-```
-
-Your certificate has been generated and is now saved on the machine. Please head over to `https://travis.example.com:8800/console/settings`.
-
-There, in the ***TLS Key & Cert*** section, select ***Server path*** and fill in the following:
-
-- SSL Private Key Filename: `/etc/letsencrypt/live/travis.example.com/privkey.pem`
-- SSL Certificate Filename: `/etc/letsencrypt/live/travis.example.com/fullchain.pem`
-
-After that, **scroll down and click "Save"**. After your changes have been saved, you can restart Travis CI Enterprise via:
-
-```
-$ replicatedctl app start
-```
-
-Let's Encrypt certificates are short-lived, this means ***they expire after 90 days.*** This means that you'll have to renew them on a regular basis. Thankfully this can be done with `certbot` as well. Run the following commands in order to renew your certificate.
-
-**Note: Be aware that this process will also introduce downtime.**
-
-```
-$ replicatedctl app stop
-$ sudo certbot renew
-$ replicatedctl app start
-```
+1. Log into your dashboard at `https://<your-travis-ci-enterprise-domain>:8800`.
+2. Click the gear icon on the rightmost side of the top menu and select 'Console Settings' from the dropdown.
```

Comment:
These are often easier to follow if you start with  the Where. 

> On the rightmost side of the top menu click the gear icon and select...

Reply:
Addressed in 192be263